### PR TITLE
spec: fix the container runtime manifest example.

### PR DESCRIPTION
**Pull request by yifan-gu**

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -765,34 +765,23 @@ JSON Schema for the Container Runtime Manifest (container manifest), conforming
...
-                "mounts": [
-                    {"volume": "work", "mountPoint": "backup"}
                 ]
-            }
+            },
+            "mounts": [
+                {"volume": "database", "mountPoint": "db"},
+                {"volume": "buildoutput", "mountPoint": "out"}
+            ],
+            "isolators": [
```
Comment (jonboulle) [inline SPEC.md]: This one isn't right, it should be within the `app`

Comment (yifan-gu) [inline SPEC.md]: @jonboulle Thanks! Lacking sleep today...

**Comment by yifan-gu** [comment]
@jonboulle Also fixed the example/container.json

**Comment by jonboulle** [comment]
LGTM, thanks!

**Pull request merged**